packages/app-bridge/src/registries/commands/CommandRegistry.ts

```diff
@@ -17,4 +18,5 @@ export type CommandRegistry = CommandNameValidator<{
     openTemplateChooser: OpenTemplateChooser;
     closeTemplateChooser: CloseTemplateChooser;
     openNavigationManager: OpenNavigationManager;
+    openNewPublication?: OpenNewPublicationPayload;
```

Comment:
Can you really open the modal without parameters?

Reply:
Actually, no. Fixed.